pkg/scheduler/plugins/proportion/proportion.go

```diff
@@ -198,6 +198,19 @@ func (pp *proportionPlugin) OnSessionOpen(ssn *framework.Session) {
 		return overused
 	})
 
+	ssn.AddJobEnqueueableFn(pp.Name(), func(obj interface{}) bool {
+		job := obj.(*api.JobInfo)
+		queueID := job.Queue
+		attr := pp.queueOpts[queueID]
+		queue := ssn.Queues[queueID]
+		pgResource := api.NewResource(*job.PodGroup.Spec.MinResources)
+		// The queue resource quota limit has not reached
+		if pgResource.Clone().Add(attr.allocated).LessEqual(api.NewResource(queue.Queue.Spec.Capability)) {
```

Comment:
We may also consider maxPodGroup number as one of resource, we can open another PR for it.

Reply:
ok